blockchain/src/index.js

```diff
@@ -324,7 +329,7 @@ app.get("/readiness", (req, res) => {
 });
 
 const loadConfig = (path) => {
-  const config = yaml.safeLoad(fs.readFileSync(path, "utf8"));
+  const config = yaml.load(fs.readFileSync(path, "utf8"));
```

Comment:
pls do not replace `load` by` safeLoad` more information [here](http://adilapapaya.com/docs/js-yaml/#safeloadstringoptions)